scripts/factory-hooks/post-settings-php/memcache.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Example implementation of ACSF post-settings-php hook.
+ *
+ * @see https://docs.acquia.com/site-factory/tiers/paas/workflow/hooks
+ */
+
+// Use ACSF feature flag to indicate if a site should use memcache backend
+if ($is_ah_env &&
+  !empty($site_settings['flags']['memcache_enabled'] &&
```

Comment:
This line has a syntax error. Should be:

`!empty($site_settings['flags']['memcache_enabled']) &&`